Buildings/Fluid/CHPs/OrganicRankine/Data/Generic.mo

```diff
@@ -0,0 +1,40 @@
+within Buildings.Fluid.CHPs.OrganicRankine.Data;
+record Generic "Generic data record for working fluid properties"
+  extends Modelica.Icons.Record;
+
+  parameter Modelica.Units.SI.ThermodynamicTemperature T[:]
+    "Thermodynamic temperature";
+  parameter Modelica.Units.SI.AbsolutePressure p[n]
+    "Saturation pressure";
+  parameter Modelica.Units.SI.Density rhoLiq[n]
+    "Density of saturated liquid";
+  parameter Modelica.Units.SI.TemperatureDifference dTRef
+    "Superheating differential temperature";
+  parameter Modelica.Units.SI.SpecificEntropy sSatLiq[n]
+    "Specific entropy of saturated liquid";
+  parameter Modelica.Units.SI.SpecificEntropy sSatVap[n]
+    "Specific entropy of saturated vapour";
+  parameter Modelica.Units.SI.SpecificEntropy sRef[n]
+    "Specific entropy of superheated vapour reference line";
+  parameter Modelica.Units.SI.SpecificEnthalpy hSatLiq[n]
+    "Specific enthalpy of saturated liquid";
+  parameter Modelica.Units.SI.SpecificEnthalpy hSatVap[n]
+    "Specific enthalpy of saturated vapour";
```

Comment:
Replace vapour with vapor throughout (use American spelling).

Reply:
Addressed in aefcd4ea80cc991d4789a1bcae5ef21f320c044e.